java-checks/src/test/java/org/sonar/java/checks/StaticMethodCheckTest.java

```diff
@@ -22,14 +22,25 @@
 import org.junit.jupiter.api.Test;
 import org.sonar.java.checks.verifier.JavaCheckVerifier;
 
+import static org.sonar.java.checks.verifier.TestUtils.nonCompilingTestSourcesPath;
+import static org.sonar.java.checks.verifier.TestUtils.testSourcesPath;
+
 class StaticMethodCheckTest {
 
   @Test
   void test() {
     JavaCheckVerifier.newVerifier()
-      .onFile("src/test/files/checks/StaticMethodCheck.java")
+      .onFile(testSourcesPath("checks/StaticMethodCheck.java"))
       .withCheck(new StaticMethodCheck())
       .verifyIssues();
   }
 
+  @Test
+  void test_non_compiling() {
+    JavaCheckVerifier.newVerifier()
+      .onFile(nonCompilingTestSourcesPath("checks/StaticMethodCheck.java"))
+      .withCheck(new StaticMethodCheck())
+      .verifyNoIssues();
```

Comment:
"no issues", really? There is a `Noncompliant` comment in this file, it should not pass!

Reply:
That was a leftover